cg/cli/workflow/balsamic/base.py

```diff
@@ -183,13 +186,13 @@ def store_housekeeper(context, case_id):
         balsamic_analysis_api.upload_bundle_housekeeper(case_id=case_id)
         LOG.info(f"Storing Analysis in ClinicalDB for {case_id}")
         balsamic_analysis_api.upload_analysis_statusdb(case_id=case_id)
-    except (BundleAlreadyAddedError, FileExistsError) as e:
-        LOG.error(f"Could not store bundle in Housekeeper and StatusDB: {e.message}!")
+    except (BundleAlreadyAddedError, FileExistsError, ValidationError) as error:
```

Comment:
ValidationErrror class doesnt have attribute message (unless you customize it!) So it will raize an error while handling an error and kill cron loop btw

Reply:
Changed